mne_bids/utils.py

```diff
@@ -544,3 +544,34 @@ def _infer_eeg_placement_scheme(raw):
         placement_scheme = 'based on the extended 10/20 system'
 
     return placement_scheme
+
+
+    def _infer_fif_split_naming(raw_fname):
```

Comment:
should we test this private function direclty?

Reply:
yeah, do we have access to the mne testing data when we run tests? if so, then there are split files there. I could probably easily get a test together to make sure the logic holds up if that is the case

Reply:
yep, we do use the testing data. That would be great !